lib/r10k/api.rb

```diff
@@ -94,18 +94,18 @@ def parse_puppetfile(io_or_content)
     #
     # @param source [Hash] A hashmap representing a control repo source (like would be defined in r10k.yaml)
     # @param branch_name [String] Branch in the control repo to build an envmap for.
-    # @param base_cachedir [String] Root of where r10k is caching things.
+    # @param cache_basedir [String] Root of where r10k is caching things.
     # @return [Hash] A hashmap representing the desired state of the environment matching the given branch in the given source.
     # @raise [NotImplementedError] Control repo source has a type that is not currently supported.
     # @raise [RuntimeError] Something bad happened, see exception message.
-    def envmap_from_source(source, branch_name, base_cachedir)
+    def envmap_from_source(source, branch_name, cache_basedir)
```

Comment:
I'm going to switch this to your preferred base_cachedir for now. We have a cleanup ticket in the next sprint where we'll switch entirely to repo_cachedir, base_cachedir, etc.